mentoring/OWNERS

```diff
@@ -5,3 +5,4 @@ approvers:
   - sig-contributor-experience-leads
 labels:
   - sig/contributor-experience
+  - area/mentorship-planning
```

Comment:
Should this label be renamed as `area/mentoring`?

`mentorship/planning` sounds good for issues but not for PRs.

Reply:
I wanted it to stand out that this was the planning/strategy and not for people looking for mentors/mentorship opportunities.

Reply:
@parispittman wdyt about this? looks like my comment and your lgtm raced :)

Reply:
> I wanted it to stand out that this was the planning/strategy and not for people looking for mentors/mentorship opportunities.

Ack.

/hold cancel